src/components/experiments/ExperimentList.jsx

```diff
@@ -33,6 +33,22 @@ export class ExperimentList extends React.Component {
           {this.props.filteredExperiments.size} / {this.props.experimentCount}{" "}
           {this.renderTitle()}
         </h1>
+        <h6>
+          <table>
+            <tbody>
+              {this.props.filteredExperiments.map(function(item, key) {
+                return (
+                  <tr key={key + 1}>
```

Comment:
Is there a reason for the `+ 1`?

Reply:
Oh hmm at one point I was displaying how many rows I was getting and wanted the count to start at 1 instead of 0. Ill remove it since I forgot I left that in, thanks!